<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QCustomPlot" name="plot_1" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>601</width>
      <height>221</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>40</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>close</string>
    </property>
   </widget>
   <widget class="QCustomPlot" name="plot_2" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>601</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="option1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>601</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>sin&amp;cos</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>tan&amp;cot</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>cosec&amp;sec</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="option2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>601</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>first</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>second</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>third</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>cosec</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>sec</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>cot</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>647</x>
     <y>67</y>
    </hint>
    <hint type="destinationlabel">
     <x>630</x>
     <y>205</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
